website/snippets/_new-sl-setup.md

```diff
@@ -13,32 +13,41 @@ Select the environment where you want to enable the Semantic Layer:
+
+This credential controls the physical access to underlying data accessed by the Semantic Layer, and all access policies set in the data platform for this credential will be respected.
 
-dbt Cloud Enterprise plans can add multiple credentials and map those to service tokens. Refer to [Add more credentials](#4-add-more-credentials) for more information.
+| Feature | Team plan | Enterprise plan |
+| --- | :---: | :---: |
+| Service tokens | Can create multiple service tokens linked to one credential. | Can create multiple service tokens linked to multiple credentials. |
+| Credentials per project | One credential per project. | Can [add multiple](#4-add-more-credentials) credentials per project. |
+| Link multiple service tokens to a single credential | ✅ | ✅ |
 
-1. In the **Set Up Semantic Layer Configuration** page, enter the credentials specific to your data platform that you want the Semantic Layer to use.
+*If you're on a Team plan and need to add more credentials, consider upgrading to our [Enterprise plan](https://www.getdbt.com/contact). Enterprise users can refer to [Add more credentials](#4-add-more-credentials) for detailed steps on adding multiple credentials.*
+
+1. After selecting the deployment environment, you should see the **Credentials & service tokens** page. 
+2. Click the **Add Semantic Layer credential** button. 
+3. In the **1. Add credentials** section, enter the credentials specific to your data platform that you want the Semantic Layer to use.
    - Use credentials with minimal privileges. The Semantic Layer requires read access to the schema(s) containing the dbt models used in your semantic models for downstream applications
    - Note, environment variables such as  `{{env_var('DBT_WAREHOUSE') }}`, aren't supported in the dbt Semantic Layer yet. You must use the actual credentials.
-<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-configure-sl.jpg" width="45%" title="Enter the credentials specific to your data platform that you want the Semantic Layer to use and select the deployment environment."/>
 
-1. Create a **Service Token** after you add the credential.
-   * Enterprise plans: Name and generate a service token on the credential page directly.
-   * Team plans: You can return to the **Project Details** page and click the **Generate a Service Token** button. 
-2. Name the token and save it. Once the token is generated, you won't be able to view this token again so make sure to record it somewhere safe.
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-add-credential.jpg" width="55%" title="Add credentials and map them to a service token. " />
+
+4. After adding credentials, scroll to **2. Map new service token**.
+5. Name the token and ensure the permission set is 'Semantic Layer Only' and 'Metadata Only'. 
```

Comment:
nit:  these are the *minimum* recommended permissions, and we do recommend you use minimal permissions, but you are not required to.  

I'd suggest replacing "is" with "includes"